sdk/assemblyscript/src/assembly/__tests__/vectors.spec.ts

```diff
@@ -104,7 +104,7 @@ it("should compute the dot product of two vectors", () => {
 it("should compute the magnitude of a vector", () => {
   const a = [1, 2, 3];
   const result = vectors.magnitude(a);
-  expect(result).toBe(sqrt<f64>(14));
+  expect(result).toBe(sqrt<f64>(14) as i32);
```

Comment:
This doesn't make much sense to me.  If you're working with floating points, then shouldn't the input be floating points?

```ts
it("should compute the magnitude of a vector", () => {
  const a = [1.0, 2.0, 3.0];
  const result = vectors.magnitude(a);
  expect(result).toBe(sqrt<f64>(14));
});
```

Reply:
im getting this error where you cant do sqrt<i32> so im forced to do it this way
https://github.com/hypermodeinc/modus/actions/runs/12059957113/job/33629495531

im not working with floating points, im working with i32s, but theres no way to take the sqrt of them directly, so i do it with f64 then convert to i32